austation/code/modules/research/techweb/all_nodes.dm

```diff
@@ -13,3 +13,13 @@
 	design_ids = list("mech_kinetic_accelerator", "hyperaoemod", "repeatermod", "resonatormod", "bountymod")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
 	export_price = 5000
+
+///////////////////////////Medical tech////////////////////
+/datum/techweb_node/cyber_organs_upgraded
+	id = "cyber_organs_upgraded"
+	display_name = "Upgraded Cybernetic Organs"
+	description = "We have the technology to upgrade him."
+	prereq_ids = list("cyber_organs")
+	design_ids = list("cybernetic_heart_u", "cybernetic_liver_u", "cybernetic_lungs_u","catcybernetic")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)
+	export_price = 5000
```

Comment:
don't override techweb nodes, please. I'm working on a way to modularize this shit soon:tm:, but until then just edit the code in the main repo.

Reply:
how _exactly_ do i do that?

Reply:
Add catcybernetic to:

https://github.com/austation/austation/blob/6aa4d1ab912d2e141a8778c8f4352e251a47306c/code/modules/research/techweb/all_nodes.dm#L530

And add `// austation -- [your witty comment about cybernetic cat ears]`  to the end of the line

Reply:
done